src/convertToObject.js

```diff
@@ -1,18 +1,15 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split('\n').slice(1).reduce((prev, item) => {
```

Comment:
Consider separating all these steps into separate variables to improve readability and code style